### PR TITLE
Init standalone app

### PR DESCRIPTION
**Pull request by illyaV**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,167 +1,249 @@
...
-  set XMLHttpRequest(XMLHttpRequest) {
-    LocalVars.XMLHttpRequest = XMLHttpRequest
-  },
+  /**
+   * @param {string|Object} applicationId|options
+   * @param {string} [secretKey]
+   * @returns {Backendless}
+   */
+  initApp(...args) {
+    const { standalone, ...restProps } = parseArgs(...args)
```
Comment (stanislav-grin) [inline src/index.js]: Where `standalone` comes from?

File: src/index.js [hunk 2]
```diff
@@ -1,167 +1,249 @@
...
+      : this
+
+    app.initProps(restProps)
+
+    app.RT.init()
+    app.Logging.reset()
+    app.Geo.Tracker.reset()
+    app.Users.setLocalCurrentUser()
 
+    // const { default: LoggingCollector } = require('./logging/collector')
```
[the remark this answers is not in the document]
Comment (illyaV) [reply src/index.js]: Yes, work is in progress

I would like to get some feedback before continue

File: src/index.js [hunk 3]
```diff
@@ -1,167 +1,249 @@
 import Request from 'backendless-request'
+import Utils from './utils'
+
+const DEFAULT_PROPS = {
+  applicationId : null,
```
[the remark this answers is not in the document]
Comment (illyaV) [reply src/index.js]: sure!

File: src/index.js (hunk 1 above)
Comment (illyaV) [reply src/index.js]: it is an option that customer will set to get a separate backendless app client

File: backendless.d.ts [hunk 4]
```diff
@@ -24,7 +24,7 @@ declare module Backendless {
      * @public
      * @type: Function
      **/
-    function initApp(applicationId: string, jsSecretKey: string): void;
+    function initApp(appId: string | Object, apiKey: string): void;
```
Comment (Valodya) [inline backendless.d.ts]: here must be an overload
```
function initApp(appId: string, apiKey: string): void
function initApp({ appId: string, apiKey: string, standalone?:boolean }): void
```

Comment (stanislav-grin) [reply backendless.d.ts]: done

**Pull request merged**